Appl/SDK_C/Levels/levels.goc

```diff
@@ -512,14 +515,23 @@ static void _pascal SetBarState (LevelsBarStates levelsBarStates);
 	GTXI_text	= "Random text for the style menus";
 	HINT_DEFAULT_FOCUS;
 	HINT_DEFAULT_TARGET;
+	/*
+	 * Set up a VisTextCharAttr chunk to specify another font and size.
+	 * This is necessary because the applying of certain styles to bitmap fonts fails.
+	 */
+	ATTR_GEN_TEXT_CHAR_ATTR = (ChunkHandle) @FooCharAttrs;
 }
 
+@chunk VisTextCharAttr FooCharAttrs = CHAR_ATTR_FONT_SIZE(FID_DTC_URW_SANS, 18);
+
 @end	Interface;
 
 /*%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 		Global Data
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%*/
 
+#define vfar  _far 

```

Comment:
Isn't this too much here? ..... compiler definition coming a bit later anyhow.

Reply:
Yes, you are right, I forgot to delete the line.